docs/api/ref/schemas/image_selected.yaml

```diff
@@ -0,0 +1,78 @@
+components:
+  schemas:
+    ImageSelected:
+      type: object
+      title: Selected image
+      description: |
+        Images selected (and possibly cropped or rotated) to show an area of interest
+        (front of product, ingredients, nutrition facts, packaging info) in a specific language.
+      properties:
```

Comment:
where is the time of image selection ? The user who selected would also be valuable.